<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="0" failures="0" finished="true" name="Test Run: 114" tests="1" total_time="Finished in: 17.38 secs."/>
<test location="/Users/phillipramirez/eclipse-workspace/Python Test/PyTesting/test/PyTesting.py" status="ok" test="testing.test_sauce_python" time="17.38">
<stdout><![CDATA[setup time
The test begins now
The connection driver before test is: <selenium.webdriver.remote.webdriver.WebDriver (session="f9fb7466f63d4eb79807685de45442f8")>
the driver at run_sauce_test begin is: <selenium.webdriver.remote.webdriver.WebDriver (session="f9fb7466f63d4eb79807685de45442f8")>
the resultant is: HEEEEEEEEEEEEEEY!
the driver first is: <selenium.webdriver.remote.webdriver.WebDriver (session="f9fb7466f63d4eb79807685de45442f8")>
the driver second is: <selenium.webdriver.remote.webdriver.WebDriver (session="f9fb7466f63d4eb79807685de45442f8")>
teardown time
]]></stdout>
<stderr/>
</test>
<launch mode="run">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="Python Test PyTesting.py"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
